code/game/objects/items/weapons/material/shield_vr.dm

```diff
@@ -0,0 +1,16 @@
+/obj/item/weapon/material/sword/shield
+	name = "makeshift shield"
+	desc = "A makeshift shield"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "riot"
+	force_divisor = 0.07
```

Comment:
if you add `edge = FALSE` below here, force calculations pull from the weight of the material and not the hardness i.e. baseball bats - you'll have to remember to adjust the force divisor too